assets/opstools/AppBuilder/classes/ABObject.js

```diff
@@ -304,8 +304,13 @@ export default class ABObject extends ABObjectBase {
 			}
 
 			// add the summary footer
-			if (summaryColumns.indexOf(f.id) > -1)
-				header.footer = { content: 'summColumn' };
+			if (summaryColumns.indexOf(f.id) > -1) {
+				if (f.key == "calculate") {
```

Comment:
Can this support "Formula" field too ?

Reply:
It worked well...I included it in this pull request as well.